core/src/main/java/com/linecorp/armeria/server/ServerBuilder.java

```diff
@@ -1502,6 +1505,30 @@ public ServerBuilder contentPreview(int length) {
         return contentPreview(length, ArmeriaHttpUtil.HTTP_DEFAULT_CONTENT_CHARSET);
     }
 
+    /**
+     * Sets whether the response header will include {@code "Server"} header.
+     */
+    public ServerBuilder disableServerHeader() {
+        this.useServerHeader = false;
```

Comment:
Especially since the method is `disable`, I think it's more readable if the variable is called `enableServerHeader` instead of `use`

Reply:
`isServerHeaderEnabled` sounds better to me.

Reply:
Renamed variables to `enableServerHeader` and `enableDateHeader`.
Thank you!